R/estimatr_difference_in_means.R

```diff
@@ -431,7 +443,9 @@ difference_in_means_internal <- function(condition1 = NULL,
 
   # Check that treatment status is uniform within cluster, checked here
   # so that the treatment vector t doesn't have to be built anywhere else
+  clustered <- FALSE
```

Comment:
clustered <- is.numeric(data$cluster)
    if(clustered) {
      ...
    }

Reply:
I won't use is.numeric because it need not be, but I can use !is.null